scripts/add_electricity.py

```diff
@@ -507,6 +547,7 @@ def attach_conventional_generators(
         efficiency=ppl.efficiency,
         marginal_cost=marginal_cost,
         capital_cost=ppl.capital_cost,
+        # Not clear how to get investment for these Generators, hence ignoring it for now
```

Comment:
```suggestion
        investment=ppl.investment,
```
I think it's already there through previous lines

```py
ppl = (
        ppl.query("carrier in @carriers")
        .join(costs, on="carrier", rsuffix="_r")
        .rename(index=lambda s: f"C{str(s)}")
)
```